actors/sunbird-lms-mw/actors/sunbird-utils/sunbird-platform-core/common-util/src/main/java/org/sunbird/common/request/UserRequestValidator.java

```diff
@@ -64,7 +64,17 @@ private void validateLocationCodes(Request userRequest) {
           ERROR_CODE);
     }
     if (locationCodes != null) {
-      List<String> set = new ArrayList(new HashSet<>((List<String>) locationCodes));
+      List<String> set = null;
+      if (((List) locationCodes).get(0) instanceof String) {
+        set = new ArrayList(new HashSet<>((List<String>) locationCodes));
+        userRequest.getRequest().put(JsonKey.LOCATION_CODES, set);
+      } else {
+        set = new ArrayList();
+        List<Map<String, String>> locationList = (List<Map<String, String>>) locationCodes;
```

Comment:
Validation for type and code need to be added

Reply:
@reshmi-nair i thought once we take school into consideration we can check all the types what all we decided.

Reply:
now we are validating the location type @reshmi-nair